### DISCUSSION TITLE
Cannot create environment for crossbar

### DISCUSSION THREAD

**Discussion by malibu1966** [crossbario/crossbar#2115] [Q&A]
Hi there, I was given 'pip install git+https@github.com/oberstet/crossbar.git@updating_dependencies' to install crossbar before, but now that is not working either.  It ends with the following:

```
Collecting eth-account@ git+https://github.com/crossbario/eth-account.git@fix-215#egg=eth-account
  Cloning https://github.com/crossbario/eth-account.git (to revision fix-215) to /tmp/pip-install-fzc09zy5/eth-account_783cb4d8726841af859dea2f1bdfe8cc
  Running command git clone --filter=blob:none -q https://github.com/crossbario/eth-account.git /tmp/pip-install-fzc09zy5/eth-account_783cb4d8726841af859dea2f1bdfe8cc
  Running command git checkout -b fix-215 --track origin/fix-215
  Switched to a new branch 'fix-215'
  branch 'fix-215' set up to track 'origin/fix-215'.
  Resolved https://github.com/crossbario/eth-account.git to commit 1aa6220b0f2c17035837f7ada4f495112f176cca
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
ERROR: Could not find a version that satisfies the requirement hexbytes>=0.3.0 (from crossbar) (from versions: 0.1.0a1, 0.1.0a2, 0.1.0b0, 0.1.0b1, 0.1.0, 0.2.0, 0.2.1, 0.2.2, 0.2.3)
ERROR: No matching distribution found for hexbytes>=0.3.0
```

Can you offer any advice?

I'm trying to promote agents into my servers at work so I need to build autobahn environments.  If I try to install just autobahn then I get ModuleNotFoundError: No module named setuptools_rust.  Any help would be appreciated, thank you.

**Comment by malibu1966** [comment]
Actually I think the crossbar install issue might be python too old.  I am only on python 3.6.  Will upgrade.

**Reply by oberstet** [reply]
> I was given 'pip install [git+https@github.com](mailto:git+https@github.com)/oberstet/crossbar.git

given by whom?

> Can you offer any advice?

use docker, eg https://hub.docker.com/r/crossbario/crossbar
